src/workerd/io/worker.c++

```diff
@@ -2951,12 +2951,13 @@ struct Worker::Actor::Impl {
     void inputGateWaiterRemoved() override { metrics.inputGateWaiterRemoved(); }
     // Implements InputGate::Hooks.
 
-    kj::Promise<void> makeTimeoutPromise() override {
+    kj::Promise<void> makeTimeoutPromise(kj::LiteralStringConst operationInfo) override {
       // This really only protects against total hangs. Lowering the timeout drastically is risky,
       // since low timeouts can spuriously fire when under heavy CPU load, failing requests that
       // would otherwise succeed.
       auto timeout = 30 * kj::SECONDS;
       co_await timerChannel.afterLimitTimeout(timeout);
+      KJ_LOG(ERROR, "Durable Object storage operation exceeded timeout", operationInfo);
```

Comment:
Can I avoid this extra log by putting the operationInfo into the exception without exposing it to the user? Is there some logic in the jsg errors that strips out information before the jsg.Error substring or something like that?

Reply:
I'm not aware of anything like that, FWIW.

We can either make make sure the `operationInfo` strings are ok to expose to users, or keep this log.

Reply:
The only thing I know about is https://github.com/cloudflare/workerd/blob/7bb684fe611d83375707b20c6c6c19bd2cfe590c/src/workerd/jsg/exception.c%2B%2B#L11, which only strips off the "remote exception" prefix.